src/popover/Popover.js

```diff
@@ -410,6 +411,36 @@ export default class Popover extends React.Component {
         anchorOffset.left = isHorizontal ? (child.width - popover.width) : 0;
         anchorOffset.top = isVertical ? (child.height - popover.height) : 0;
         break;
+      case 'auto':
+        _realAnchor = 'start';
+        if (isHorizontal) {
+          const scrollX = (window.pageXOffset || document.scrollLeft || 0) - (document.clientLeft || 0);
```

Comment:
The scroll of the `document` (otherwise we would have also the element's scrollLeft as parameter of the operation)